docs/guide/nested-module-subdirs.md

```diff
@@ -0,0 +1,113 @@
+# Nested module folders
+
+Apostrophe has optional support for nested subdirectories of modules, tucked
+inside `modules`.
+
+You must set the `nestedModuleSubdirs` option to `true` in `app.js`, like this:
+
+
+<AposCodeBlock>
+```javascript
+require('apostrophe')({
+  shortName: 'my-project',
+  nestedModuleSubdirs: true
+  // etc., you may have additional options here as always
+  modules: {
+    // You can still enable all modules here, or move some or all to modules.js
+    // files in subdirectories, as seen below
+  }
+});
+```
+  <template v-slot:caption>
+    app.js
+  </template>
+</AposCodeBlock>
+
+Now you can nest modules in subdirectories, like the example `modules/products` folder explored below. Start with a `modules.js` file in the parent `modules/products` folder. Here you can activate all of the modules that relate to products, making `app.js` shorter:
```

Comment:
This would be the place to say that the `products` directory name is arbitrary. It could even be `sales-modules` to be extra distinct.